### ISSUE TITLE
NVDA does not announce volume and brightness level changes when focus is not in an explorer.exe window.

### ISSUE THREAD

**Issue by mazen428** [nvaccess/nvda#13197]
### Steps to reproduce:
Launch any program, Settings for example.
Press the volume or brightness keys, usually they are something between FN + F 1 to F 12.
### Actual behavior:
NVDA says nothing
### Expected behavior:
NVDA reports the new volume or brightness level like in explorer windows such as the taskbar or the desktop
#### NVDA installed/portable/running from source:
Installed.
#### NVDA version:
2021.3
#### Windows version:
Windows 11 21 h2, I think this also happens on Windows 10 too but I'm not sure.
#### Name and version of other software in use when reproducing the issue:
Any.
#### Does the issue still occur after restarting your computer?
Yes.
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
2021.2, same tissue.
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes.
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes.

**Comment by mazen428** [comment]
Oh, I just saw this issue, I actually thought it's a bug and was confused on why.
https://github.com/nvaccess/nvda/issues/10626
Sorry. Will close.

**Comment by comanna** [comment]
Hi,

 

I am trying to access a document om the web using the latest Firefox in Windows 10 and the latest NVDA.

 

After activating the “Submit” button on the web page, An alert notice appears, which, I understand, is telling me that Firefox has blocked pop-ups. However All NVDA tells me in “options button” and “Close button” But no report of the message. Object Review and Screen review fail to move to any other information on the screen.

 

By pressing NVDA +b I can read the entire screen and all  its exhausting detail before the very last line appears with the text of the message that I should have read immediately.

 

Is there some setting that I can select to make NVDA read the alert itself so that I don’t have to read the entire screen?

 

All the best,

 

Cearbhall

 

m +353 (0)833323487 Ph: _353 (0)1-2864623 e: ***@***.***



-- 
This email has been checked for viruses by Avast antivirus software.
https://www.avast.com/antivirus

**Comment by cary-rowen** [comment]
As discussed in #10626 when we adjust the volume in other applications, can we play a short sound to inform the user of the current volume?

**Comment by josephsl** [comment]
Hi, @LeonardDer, I know we discussed this before, but is there a justification to reconsider focus checks when handling UIA notification event in 2022 so background notifications can be handled or make it configurable somehow? Thanks.

**Comment by josephsl** [comment]
Hi,

As for playing a sound to indicate system volume when it changes, this will require adding UIA notification event handler to File Explorer app module to enable this possibility from the background i.e. when desktop or File Explorer is not in focus. What makes this a bit complicated is a possibility that UIA notification properties might differ between Windows releases, especially activity Id which is used to identify what the notification is for when raised by apps.

Thanks.

**Comment by Brian1Gaff** [comment]
Please make this into a new issue, not piggy back it onto an existing one, 
thanks.
 Brian

***@***.***
Sent via blueyonder.
Please address personal E-mail to:-
***@***.***, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users
----- Original Message -----

**Comment by leonardder** [comment]
I find these announcements pretty disturbing for my workflow, so I don't think we should reconsider this for a change in default behavior. We could make it more configurable, though.
